plugins/inputs/mongodb/README.md

```diff
@@ -10,6 +10,7 @@
   ##   mongodb://10.10.3.33:18832,
   ##   10.0.0.1:10000, etc.
   servers = ["127.0.0.1:27017"]
+  gather_db_stats = false
```

Comment:
that is incorrectly labelled in the changelog, sorry about that, it's not available yet